server_config/settings.json

```diff
@@ -3,7 +3,7 @@
     "phantomasEngine": "webkit",
     "googleAnalyticsId": "",
     "screenshotWidth": 400,
-
+    "baseUrl": "/",
```

Comment:
Could you please add the same line in /test/fixtures/settings.json? The Travis CI tests are broken just because of this missing line.